main.py

```diff
@@ -83,7 +83,7 @@
 from core.platform import models
 from core.platform.auth import firebase_auth_services
 
-import google.cloud.logging
```

Comment:
Is this an expected change?

Reply:
it was not in use. so for that reason on line 107
function is called like this
```
 client = google.cloud.logging.Client()
```

Reply:
should I invert it

Reply:
Yup, let's revert this change.